pkg/target/manager.go

```diff
@@ -30,9 +30,13 @@ type Manager interface {
 	CurrentTarget() (Target, error)
 
 	TargetGarden(name string) error
+	DropTargetGarden() (string, error)
 	TargetProject(ctx context.Context, name string) error
+	DropTargetProject() (string, error)
 	TargetSeed(ctx context.Context, name string) error
+	DropTargetSeed() (string, error)
 	TargetShoot(ctx context.Context, name string) error
+	DropTargetShoot() (string, error)
```

Comment:
it's maybe better readable if we group the functions by `Target..` and `DropTarget..`

Reply:
I grouped the interface methods. Shall we also group the implementation functions?

Reply:
yes makes sense